File: Lua_5.4/2150 - Vogais Alienígenas.lua
--- author: Josélio de S. C. Júnior <joseliojrx25@gmail.com>
--- copyright: Josélio de S. C. Júnior 2021

local a = {}
while true do
  local vowels = io.read()
  if #vowels < 1 then break end
  local text = io.read()
  table.insert(a, #text:gsub('[^' .. vowels .. ']', ''))
end
print(table.concat(a, '\n'))

File: JavaScript_12.18/2150 - Vogais Alienígenas.js
/**
 * @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
 * @copyright Josélio de S. C. Júnior 2021
*/

const p = require('fs').readFileSync('/dev/stdin', 'utf8');
const l = p.split('\n');

const a = [];
while (true) {
    const vowels = l.shift();
    if (!vowels) break;
    const text = l.shift();
    const regex = new RegExp(`[^${vowels}]`, 'g');
    a.push(text.replace(regex, '').length);
}
console.log(a.join('\n'));